### DISCUSSION TITLE
V2 Yarn mappings for 3D Shareware v1.34 do not exist

### DISCUSSION THREAD

**Discussion by ManosSef** [FabricMC/fabric-api#3672] [Mod Dev Support] [answered]
### Description

I'm trying to make a mod for 3D Shareware v1.34 (don't ask) and Loom is trying to find Yarn mappings with the filename "yarn-3D Shareware v1.34.2-v2.jar", which doesn't exist.
![image](https://github.com/FabricMC/fabric/assets/65536780/acf0d1c0-40de-4322-a2a8-0b6d37196bcf)
![error](https://github.com/FabricMC/fabric/assets/65536780/b4826e77-d794-4ebb-8257-7df065d5d739)
Every version of Minecraft that's supported by Fabric has a v2 version of the Yarn mappings, except for this one. I've tried editing build.gradle so that it doesn't search for the v2 mappings and instead searches for the original ones, but that caused a very generic error saying it "Failed to remap minecraft."
![other_error](https://github.com/FabricMC/fabric/assets/65536780/e11c1c5c-2716-4eec-b739-4523331be9bf)
So how can I make a mod project for 3D Shareware v1.34?

**Comment by tildejustin** [comment]
hopefully this acceptable self-promotion, I updated the 3d shareware mappings a while ago to make them usable with modern loom, <https://github.com/tildejustin/yarn/tree/3D_Shareware_v1.34>.

**Reply by ManosSef** [reply]
It seems that it cannot find any of your files. Maybe you've changed something since then? I followed the instructions on your github to the letter.
![cant find them](https://github.com/FabricMC/fabric/assets/65536780/3af98882-c0a4-4c23-9199-e9e7424061b3)

**Reply by tildejustin** [reply, marked as answer]
for mine you should still use the v2 classifier, ex. `dev.tildejustin:yarn:3d-shareware-v2:v2`. jitpack just rebuilt the artifact for some reason, seems to work now.

**Reply by ManosSef** [reply]
Thanks so much! It works now.

**Comment by modmuss50** [comment]
The V2 mappings are broken, just remove the `:v2` classifier from your build.gradle and it should work.

**Reply by ManosSef** [reply]
I've already mentioned that I tried that and it didn't work, I got the "Failed to remap minecraft" error.

**Comment by thedirptastic** [comment]
How did you get the verson to work? I have the mappings working, but the version gives me the error "Could not find net.fabricmc:intermediary:3D%20Shareware%20v1.34"

**Reply by tildejustin** [reply]
I just ran into that issue a couple weeks ago, haha. Putting `setIntermediaryUrl("https://maven.fabricmc.net/net/fabricmc/intermediary/%s/intermediary-%1\$s-v2.jar")` in the loom block in build.gradle fixed it for me, alternatively you could downgrade to loom 1.5.
